tests/apps/actorfeatures/app.go

```diff
@@ -0,0 +1,345 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/gorilla/mux"
+)
+
+const appPort = 3000
+
+const actorMethodURLFormat = "http://localhost:3500/v1.0/actors/%s/%s/%s/%s"
+
+const registedActorType = "testactorfeatures" // Actor type must be unique per test app.
+const actorIdleTimeout = "1h"
+const actorScanInterval = "30s"
+const drainOngoingCallTimeout = "30s"
+const drainBalancedActors = true
+const secondsToWaitInMethod = 5
```

Comment:
nit: why not use multiple line const syntax ? I would use the same style with the existing code.

Reply:
Will do.